### ISSUE TITLE
Callback on timer

### ISSUE THREAD

**Issue by Ulrar** [dom96/SimpleIRC#26]
Hi,

Besides binding a function on Ping, is there a way to have some arbitrary function be called once in a while ? I need to check on an external API for notifications (to send them on the irc channel), and I don't want to put that in PrivMsg since that would require people to be active for it to work.
I don't see a way to get the parameters for sendMsg outside of a callbacks, so a timer would be great.

**Comment by dom96** [comment]
IIRC SimpleIRC uses threads so I think you can also create your own thread to perform some action every Night seconds.

**Comment by Ulrar** [comment]
Yeah but as mentioned, I don't see a way to get the server from that.
I ended up using onNumeric to start a thread at connection, and passing the server from there, but that doesn't really look good I think, there must be a "proper" way to do it.
